app/models/blog.rb

```diff
@@ -10,7 +10,7 @@ class Blog < ApplicationRecord
   scope :published, -> { where('secret = FALSE') }
 
   scope :search, lambda { |term|
-    where("title LIKE '%#{term}%' OR content LIKE '%#{term}%'")
+    where('title LIKE ? OR content LIKE ?', "%#{term}%", "%#{term}%")
```

Comment:
```suggestion
    where('title LIKE :hoge OR content LIKE :hoge', ...)
```

みたいにキーワードを指定すると`"%#{term}%"`を2回書かずに済みます。

あと、LIKE検索の特殊文字をエスケープする便利メソッドもあるので、それについても調べてみてください〜。